apps/meteor/app/lib/server/lib/validateCustomMessageFields.ts

```diff
@@ -0,0 +1,41 @@
+import Ajv from 'ajv';
+import mem from 'mem';
+
+const ajv = new Ajv();
+
+const customFieldsValidate = mem((customFieldsSetting: string) => {
```

Comment:
Just checking if its good to have this `mem` cached forever (default value of maxAge is infinity)

Reply:
This would also leave the memoized return value there forever as mem won't clean it 👀

Reply:
Why not? Since it's based on the key (schema)

Reply:
I mean, the value will be valid depending on the cache and params.
 
What I was referring is that whn u don't pass a maxAge to mem, it doesn't cleanup the cache. So every time the setting value is changed and the new value is set on the cache, the old value will be just there. 

```
if (typeof maxAge === 'number') {
		// TODO: Drop after https://github.com/SamVerschueren/map-age-cleaner/issues/5
		// @ts-expect-error
		mapAgeCleaner(cache);
	}
```

Reply:
Ok, got it, you are worried about memory consumption. I didn't think about that. Just sent a fix, thanks @KevLehman